testpmd/hooks/roles/apply-cnf-cert-config/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+app_ns: "{{ dci_openshift_app_ns|default('example-cnf') }}"
+operators_regexp: "{{ tnf_operators_regexp|default('cnf-app-mac|testpmd-operator') }}"
+cnfs_regexp: "{{ tnf_cnfs_regexp|default('testpmd-operator|testpmd-app') }}"
```

Comment:
I'm not sure if these default values are accurate, `testpmd-operator` is not used in any name of a pod. 

Ideally, for `example-cnf`, the filter should be done in the labels rather than the names, these are the types used by each pod in `example-cnf`:

- example-cnf-type=lb-app
- example-cnf-type=cnf-app
- example-cnf-type=pkt-gen-app
- example-cnf-type=testpmd-operator
- example-cnf-type=trex-operator
- example-cnf-type=pkt-gen

At least the filtering should be on those pods that use CNFs under, what I *think*, are these names:

- testpmd-app-*
- loadbalancer-*

https://www.distributed-ci.io/files/4cc871c4-bb57-42b0-ac1c-6eb4bc4b0997